packages/plugins/plugin-firebase/src/methods/identify.ts

```diff
@@ -6,6 +6,27 @@ export default async (event: IdentifyEventType) => {
     await firebaseAnalytics().setUserId(event.userId!);
   }
   if (event.traits) {
-    await firebaseAnalytics().setUserProperties(event.traits as any);
+    let eventTraits = event.traits;
+    let safeTraits: Record<string, string>;
+
+    const checkType = (value: any) => {
+      return typeof value === 'object' && !Array.isArray(value);
```

Comment:
Shouldn't it be `typeof value === 'object' || Array.isArray(value);` ? so that it returns if the variable is array or object?